test/unittest/source/test_http.cpp

```diff
@@ -116,12 +117,29 @@ http {
                   
                } // configuration
 
+               void block_until_inbound_ready()
+               {
+                  constexpr auto curl = R"(curl -s -X POST -d 'ping' -H "Content-Type: text/plain" http://localhost:7042/casual/example/echo)";
+
+                  for( int i = 0; i < 1000; ++i)
+                  {
+                     auto capture = administration::unittest::cli::command::execute( curl);
+                     if( capture)
+                        return;
+                     
+                     common::process::sleep( std::chrono::milliseconds{ 10});
+                  }
+
+                  common::code::raise::error( code::casual::invalid_semantics, "failed to get a response from inbound after 1000 attempts");
+               }
```

Comment:
Use `unittest::eventually::succeed` instead, and just do a lambda that returns false until it succeed.